modules/operators-overview.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='operators-overview-{context}']
+= Operators in {product-title}
+
+In {product-title} version 4.0, all cluster functions are divided into a series
+of component Operators.
+
+Operators can manage other Operators within their component. This lends to a
+transparent model: a single Operator usually manages a single binary file.
```

Comment:
No, that’s not really true (in fact it’s the opposite).

An operator provides users (admins or developers) with a simple API for expressing the desired cluster functionality.  The operator hides the details of managing the lifecycle of that component.  Operators may manage a single component or tens of components, but the end goal is always to reduce operational burden by automating common actions.